challenge5/index.html

```diff
@@ -0,0 +1,292 @@
+
+        <div class="flex flex-col rounded-2xl overflow-hidden bg-white">
+          <div class="w-full">
+            <img src="./assets/imgs/image2.svg" alt="Illustration" class="w-full" />
+          </div>
+          <h3 class="text-2xl font-bold pt-6 pb-3 pl-8 pr-16">
+            Herbal medicines that are safe for consumption
+          </h3>
+          <p class="text-lg pb-6 pl-8 pr-12 font-light text-gray-400">
+            Herbal medicine is very widely used at this time because of its
+            very good for your health...
+          </p>
+          <div class="flex flex-row mt-auto pl-8 pr-12 pb-8 hover:opacity-80">
+            <a href="#" class="text-secondary_color text-lg font-medium pr-3">Read more</a>
+            <img src="./assets/icons/arrow.svg" alt="Arrow" />
+          </div>
+        </div>
+
+        <div class="flex flex-col rounded-2xl overflow-hidden bg-white">
+          <div class="w-full">
+            <img src="./assets/imgs/image3.svg" alt="Illustration" class="w-full" />
+          </div>
+          <h3 class="text-2xl font-bold pt-6 pb-3 pl-8 pr-16">
+            Natural care for healthy facial skin
+          </h3>
+          <p class="text-lg pb-6 pl-8 pr-12 font-light text-gray-400">
+            A healthy lifestyle should start from now and also for your skin
+            health. There are some...
+          </p>
+          <div class="flex flex-row mt-auto pl-8 pr-12 pb-8 hover:opacity-80">
+            <a href="#" class="text-secondary_color text-lg font-medium pr-3">Read more</a>
+            <img src="./assets/icons/arrow.svg" alt="Arrow" />
+          </div>
+        </div>
+      </div>
+
+      <div class="flex self-center justify-center pt-7 md:pt-12">
+        <button class="bg-primary_color text-secondary_color rounded-full text-button px-10 w-button border-solid border-2 border-secondary_color self-center hover:opacity-80">
+          View all
+        </button>
+      </div>
+    </section>
+  </main>
+  <footer class="bg-gradient-to-t from-top_gradient to-bottom_gradient">
+    <div class="flex flex-col md:flex-row md:items-start pr-14 pl-20 py-10">
+      <div class="md:w-2/5 flex flex-col items-start">
+        <div class="flex items-center mr-4">
+          <div class="w-10 h-10 bg-blue-500 rounded-full flex items-center justify-center">
+            <span class="text-white font-bold text-lg">T</span>
+          </div>
+          <span class="ml-3 text-white font-bold text-xl">Trafalgar</span>
+        </div>
+        <p class="text-white font-light pt-5 pb-4">
+          Trafalgar provides progressive, and affordable healthcare,
+          accessible on mobile and online for everyone
+        </p>
+        <p class="text-white font-light pt-5 pb-4">
+          ©Trafalgar PTY LTD 2020. All rights reserved
+        </p>
+      </div>
+      <div class="pt-12 md:pt-0 md:pl-20 pr-40 text-white">
+        <h3 class="font-bold">Company</h3>
+        <ul class="text-white pt-5">
```

Comment:
We should use gaps instead of paddings when possible. In this case we can set a gap at the parent div.

If we are forced to place the spacing in the `ul`, I think margin would be more appropriate here to add space between the `ul` and the `h3`, since padding is internal to the `ul` element while margin isn't.